### ISSUE TITLE
Prismatik stopped capturing Netflix & HboGO in Opera. Youtube works fine

### ISSUE THREAD

**Issue by theyv** [psieg/Lightpack#501]
In chrome both netflix and hbo go works fine. I tried to change capture source in settings with no luck. 
Any possibilities to fix this?

cheers

**Comment by six-shooter** [comment]
For me, it doesn't work in chrome. have not tried any other browsers

**Comment by FluffyDiscord** [comment]
its DRM, probly no way around it, no screen capture works, you would need to capture it with some capturing card/device in the middle

**Comment by psieg** [comment]
Yeah, because of DRM it's actively being blocked from being captured. Maybe one of the suggestions in #470 works?

**Comment by FluffyDiscord** [comment]
#470 doesn't work any longer so that's it. The "workaround" is to download your movie/series and watch it offline in your own player

**Comment by theyv** [comment]
@six-shooter @psieg @FluffyDiscord 
i figured some "workaround"
With this chrome version:` 79.0.3945.117`
netflix works fine.
Sadly HBO MAX doesn't work (says "unsupported browser"), but with this user agent:
`Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.82 Safari/537.36`
it works fine.
But we can't change user agent with extensions, it would not work. You must do it this way:

> 1. Right Click Anywhere in Webpage > Inspect (Alternatively, you can use CTR+Shift+I on Windows, Cmd + Opt +J on Mac.)
> 2. Choose More Tools > Network Conditions
> 3. Uncheck Select Automatically Checkbox
> select custom and type useragent I given you (or any newest, just check it here if it not work https://www.whatismybrowser.com/guides/the-latest-user-agent/chrome)
> go to https://play.hbomax.com/ (make sure that you are not refreshing the "unsupported device" subpage)

**Comment by jorgemario7** [comment]
I had the same problem and fix it by downloading Netflix app and updating drivers using outbyte driver updater. The app is better you will get higher video resolution compared to the browser and also surround sound, something that browsers lack.

**Comment by theyv** [comment]
@jorgemario7
for me the netflix windows app doesn't work with prismatik :(


wt., 5 kwi 2022 o 06:02 jorgemario7 ***@***.***> napisał(a):

**Comment by jorgemario7** [comment]
I followed this tutorial to install and configure Arduino and Prismatik

Video Tutorial
https://www.youtube.com/watch?v=juC-3imLoTQ&t=266s

Adalight Code

https://github.com/Wifsimster/adalight_ws2812

I used to have arduino nano Chinese version, it gave me a lot of issues and it stop working so I got the original and I am happy with it and works better.  

Install FTDI drivers

https://ftdichip.com/drivers/vcp-drivers/


Update all drives by going to your manufacturers website and under support - drivers  - scan your system and update your pc. 
 
Run windows update 

Since I build my own pc I used Outbyte Driver updater to update my system. 

When it stops working or have issues  
I uninstall (do not delete) 
Usb serial converter
Usb 3.0 (if you are using usb 3 to connect your arduino)  
Usb device Descriptor Request Failed 
No need to close Prismatik, do not reboot pc

Disconnect arduino usb from pc
Unplug arduino  power supply.
Wait few seconds and reconnect arduino power supply and reconnect usb to pc
Let pc install the best drivers
for you pc. This works for windows 10 & 11. 

I hope this helps
